concepts/booleans/about.md

```diff
@@ -1 +1,23 @@
-TODO about: booleans
\ No newline at end of file
+Booleans in Haxe are represented by the `Bool` type, whose values can be either `true` or `false`.
+
+Haxe supports three [boolean operators][operators]: `!` (unary NOT), `&&` (binary AND), and `||` (binary OR)
+
+Compound boolean expressions (`&&` and `||`) use _short-circuit_ evaluation, where expressions are evaluated from left to right but only as far as necessary. This is significant in cases such as the following:
+
+```haxe
+if (object != null && object.field == x) {}
```

Comment:
Looking at the prerequisites of this exercise, it looks like students might not be familiar with `if` statements. As such, maybe rewrite this to:

```suggestion
object != null && object.field == x
```